test/e2e/storage/testsuites/provisioning.go

```diff
@@ -725,6 +725,28 @@ func StopPodAndDependents(c clientset.Interface, pod *v1.Pod) {
 	} else {
 		framework.Logf("Pod %s has the following logs: %s", pod.Name, body)
 	}
+
+	// We must wait explicitly for removal of the generic ephemeral volume PVs.
+	// For that we must find them first...
+	pvs, err := c.CoreV1().PersistentVolumes().List(context.TODO(), metav1.ListOptions{})
+	framework.ExpectNoError(err, "list PVs")
+	var podPVs []v1.PersistentVolume
+	for _, pv := range pvs.Items {
+		if pv.Spec.ClaimRef == nil ||
+			pv.Spec.ClaimRef.Namespace != pod.Namespace {
```

Comment:
How do we know these PVs are generic ephemeral volumes and not just normal PVs?

Reply:
I guess either way, we want all PVs to be cleaned up regardless.

Reply:
> How do we know these PVs are generic ephemeral volumes and not just normal PVs?

Because they are bound to the PVCs which are owned by the pod that is being deleted.